src/stan-users-guide/regression.Rmd

```diff
@@ -25,24 +27,15 @@ $$
 y_n \sim \operatorname{normal}(\alpha + \beta X_n, \, \sigma).
 $$
 
-This latter form of the model is coded in Stan as follows.
+This latter form of the model is coded in Stan as follows [linear_regression_1](https://github.com/stan-dev/docs/tree/footnote/src/stan-users-guide/linear_regression_1)
```

Comment:
1. `footnote/src/stan-users-guide/linear_regression_1` should be part of this PR.

2.  Shouldn't this be pointing to what is called `linear_regression_1/regression_1.stan` below?

3.  This won't be gramatical when rendered---it'll say 

> This latter form of the model is coded in stan as follows [linear_regression_1](foo.html)

It's missing connectors and a final sentence period.  I'd suggest coding it so it renders as:

> This latter form of the model is coded in stan as follows (see [source code](foo.html)).

You can look at the source of this comment to see how it needs to be coded.

Reply:
> 

> 1. `footnote/src/stan-users-guide/linear_regression_1` should be part of this PR.
Did I screw up? Do you not see:https://github.com/stan-dev/docs/tree/footnote/src/stan-users-guide/linear_regression_1 ? 

> 2. Shouldn't this be pointing to what is called `linear_regression_1/regression_1.stan` below?
No, we want to point to the directory that contains code that has interface programs that can run the .stan program. If we link straight to the .stan file the other programs are not obvious.

> 3. This won't be gramatical when rendered---it'll say
> 
> > This latter form of the model is coded in stan as follows [linear_regression_1](foo.html)
> 
> It's missing connectors and a final sentence period. I'd suggest coding it so it renders as:
> 
> > This latter form of the model is coded in stan as follows (see [source code](foo.html)).
> 

How about something like: This latter form of the model is coded in stan as follows (see [interface/source code](foo.html)).